docs/quick-start/write-data.txt

```diff
@@ -0,0 +1,97 @@
+.. _laravel-quick-start-write-data:
+
+=================================
+Post an API Request to Write Data
+=================================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create the Function to Store Data
+
+      Add logic to save the data from a request in the ``MovieController.php``
+      file in the ``app/Http/Controllers/`` directory. Replace the existing
+      placeholder ``store()`` method with the following code:
+
+      .. code-block:: php
+
+         public function store(Request $request)
+         {
+             $data = $request->all();
+             $movie = new Movie();
+             $movie->fill($data);
+             $movie->save();
+         }
+
+   .. step:: Add a Route for the Controller Function
+
+      Add the API route to map to the ``store()`` method in the
+      ``routes/api.php`` file.
+
+      .. code-block:: php
+
+         use App\Http\Controllers\MovieController;
+
+         // ...
+
+         Route::resource('movies', MovieController::class)->only([
+           'store'
+         ]);
+
+
+   .. step:: Update the Movie Model
+
+      Add the movie detail fields to the ``$fillable`` array
+      of the ``Movie`` model. After adding them, the ``Movie``
+      class at ``app/Models/Movie.php`` should contain the
+      following code:
```

Comment:
S: I think the wording of this step can be more action-oriented:

```suggestion
      Navigate to the ``Movie`` class located at ``app/Models/Movie.php``. Then, add
      the following movie detail fields to the ``$fillable`` array of the ``Movie`` model:
```

Reply:
Thanks for the suggestion. I think the current wording is action-oriented, but might be a little confusing since it states what the code should look like after updating it. This is a pattern used in other steps of the Quick Start, but I'll reword this to possibly make it easier to read.